src/polymer/polymer2-element-scanner.ts

```diff
@@ -303,20 +311,37 @@ class ElementVisitor implements Visitor {
 
   private _getObservedAttributes(node: estree.ClassDeclaration|
                                  estree.ClassExpression) {
+    const returnedValue =
+        this._getReturnValueOfStaticGetter(node, 'observedAttributes');
+    if (returnedValue && returnedValue.type === 'ArrayExpression') {
+      return this._extractAttributesFromObservedAttributes(returnedValue);
+    }
+  }
+
+  private _getObservers(node: estree.ClassDeclaration|estree.ClassExpression) {
+    const returnedValue = this._getReturnValueOfStaticGetter(node, 'observers');
+    if (returnedValue) {
+      return extractObservers(returnedValue, this._document);
+    }
+  }
+
+  private _getReturnValueOfStaticGetter(
+      node: estree.ClassDeclaration|estree.ClassExpression,
+      methodName: string) {
```

Comment:
These getters don't have explicit return types in the function declarations.  Is that a style choice / because these are private and typescript is smart enough?